Application/EdFi.Ods.AdminApi.AdminConsole/Infrastructure/DataAccess/Artifacts/Admin/MsSql/20250120152308_AddApiCredentialsToInstance.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EdFi.Ods.AdminApi.AdminConsole.Infrastructure.DataAccess.Artifacts.Admin.MsSql
+{
+    /// <inheritdoc />
+    public partial class AddApiCredentialsToInstance : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "ApiCredentials",
+                schema: "adminconsole",
+                table: "Instances",
+                type: "nvarchar(max)",
+                nullable: true
+             );
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
Can you implement the `Down` on this migration. I think all we have to do is delete the new column.